### PR TITLE
Update update-data.py

### PR DESCRIPTION
**Pull request by Rubber1Duck**
Minor improvements. Remove not nessesary "If" statements. its my first pull request! I hope everything is right!

### CODE REVIEW

File: bin/update-data.py [hunk 1]
```diff
@@ -97,39 +97,36 @@ def load_ecdc_diff_data(result, dates):
...
     print('Reading ECDC daily diff data ...')
-  diff_data = None
   try:
     diff_data = pd.read_csv('https://opendata.ecdc.europa.eu/covid19/casedistribution/csv')
     diff_data.drop(['day', 'month', 'year', 'geoId', 'countryterritoryCode', 'popData2019'], axis=1, inplace=True)
     diff_data.to_csv(os.path.join(src_path, 'ecdc-casedistribution.csv'), index=False)
   except urllib.error.URLError as e:
     print(e, file=sys.stdout)
     print('Falling back to locally cached ECDC data ...')
-  if diff_data is None:
```
Comment (607011) [inline bin/update-data.py:108]: I don't like an alternative action taking place while handling an exception.

File: bin/update-data.py [hunk 2]
```diff
@@ -97,39 +97,36 @@ def load_ecdc_diff_data(result, dates):
...
     diff_data = pd.read_csv(os.path.join(src_path, 'ecdc-casedistribution.csv'))
-  if diff_data is not None:
```
Comment (607011) [inline bin/update-data.py:110]: `pd.read_csv()` from a file can also fail, e.g. if the file doesn't exist yet. Thus its necessary to handle this case.

File: bin/update-data.py [hunk 3]
```diff
@@ -97,39 +97,36 @@ def load_ecdc_diff_data(result, dates):
   dt = latest_day - first_day
   if verbosity > 0:
     print('Reading ECDC daily diff data ...')
-  diff_data = None
```
Comment (607011) [inline bin/update-data.py:100]: You're right, it's unnecessary. But I like to keep it for clarity.

File: bin/update-data.py (hunk 1 above)
Comment (Rubber1Duck) [reply bin/update-data.py:108]: Understand this ...

File: bin/update-data.py (hunk 2 above)
Comment (Rubber1Duck) [reply bin/update-data.py:110]: But the File is part of the Repository and should never not exist! But also understand this

File: bin/update-data.py (hunk 3 above)
Comment (Rubber1Duck) [reply bin/update-data.py:100]: Okay, take my pull request, and put it in the Tonne :-)

Comment (Rubber1Duck) [reply bin/update-data.py:100]: or must i delete this request? but how?

**Pull request closed without merging**